src/one/nio/serial/GeneratedSerializer.java

```diff
@@ -39,12 +39,15 @@ public class GeneratedSerializer extends Serializer {
     static final AtomicInteger renamedFields = new AtomicInteger();
     static final AtomicInteger unsupportedFields = new AtomicInteger();
 
+    private final boolean jsonOnlySerialization;
```

Comment:
What do you think about creating separate JsonSerializer with own configuration ?
It seems that json serialization can be extended in the fututre and it whould be to heavy for java serializer

Reply:
Great idea. I'll see what I can come up with.